scripts/deploy.py

```diff
@@ -154,6 +156,30 @@ def preprocess_release():
             shutil.copyfile(src, dst)
 
 
+def _get_served_version():
+    """Retrieves the default version being served on the specified application
+    being served on app engine.
+
+    Returns:
+        (str): The current serving version.
+    """
+    listed_versions = subprocess.check_output(
+        [APPCFG_PATH, '--application=%s' % APP_NAME, 'list_versions'])
+    listed_versions = listed_versions[listed_versions.index('default: [') + 10:]
```

Comment:
Consider pulling out string constant and using len(string constant) instead of 10